src/main/java/hudson/plugins/ec2/SlaveTemplate.java

```diff
@@ -170,7 +174,7 @@
     public SlaveTemplate(String ami, String zone, SpotConfiguration spotConfig, String securityGroups, String remoteFS,
             InstanceType type, boolean ebsOptimized, String labelString, Node.Mode mode, String description, String initScript,
             String tmpDir, String userData, String numExecutors, String remoteAdmin, AMITypeData amiType, String jvmopts,
-            boolean stopOnTerminate, String subnetId, List<EC2Tag> tags, String idleTerminationMinutes,
+            boolean stopOnTerminate, String subnetId, List<EC2Tag> tags, String idleTerminationMinutes, String minimumNumberOfInstancesStr,
```

Comment:
Are you able to use `int` or `Integer` here?

Reply:
It should be possible. I just followed the pattern of existing "int-type" parameters like `numExecutors` and `idleTerminationMinutes` where a String is parsed instead. 

Not sure what the reasoning behind those are since I'm not too familiar with the Jenkins gui code. And at any rate, form validation occurs before posting.

Reply:
Should I change to `int`?

Reply:
Yeah, just stick to int where you can here as long as it works. The other variables are probably legacy.

Reply:
Could you fix this bit @erikhakansson

Reply:
Sorry, I haven't had time to sit with this for a while but I'm looking into it now!

Reply:
Done :)